### ISSUE TITLE
Discriminator loss function differs from paper?

### ISSUE THREAD

**Issue by davidstap** [taoxugit/AttnGAN#48]
Can someone explain what the function of `cond_wrong_errD` is in the discriminator loss function? It seems not a part of the discriminator loss mentioned in the paper (Eq. 5). Also, it does not make sense to me. Why ignore the last entry in the batch?

```
cond_wrong_logits = netD.COND_DNET(real_features[:(batch_size - 1)], conditions[1:batch_size])
cond_wrong_errD = nn.BCELoss()(cond_wrong_logits, fake_labels[1:batch_size])
```

**Comment by ShihuaHuang95** [comment]
unmatched loss

**Comment by awkrail** [comment]
Hi, I met the same problem. Is this loss unmatched loss written in the paper?
